app/webpacker/controllers/order_info_controller.js

```diff
@@ -3,6 +3,24 @@ import { Controller } from "stimulus"
 export default class extends Controller {
   static values = { resource: String, shipmentNumber: String }
 
+  connect() {
```

Comment:
I'm also wondering if handling this behavior should be better in another controller such as `toggle_class` and you can parametrized on which event you want to toggle which class. This could be tested then.

Reply:
I'm not sure I follow, Do you mean we should extract the highlighting functionality when user hovers on the `edit/delete` btns to another controller?

Reply:
Yes, exactly! A `add_class` controller that use `target` and `value` to be as agnostic as possible around element, class, events to be used, but highly configurable and testable ;)

Reply:
ah! I get it. Ok will change this accordingly